src/lib/grpc-devtools.ts

```diff
@@ -0,0 +1,87 @@
+// Taken from this comment on github:timostamm/protobuf-ts:
+// https://github.com/timostamm/protobuf-ts/issues/584#issuecomment-1735664168
+
+import { RpcError, type RpcInterceptor } from "@protobuf-ts/runtime-rpc";
+
+const type = "__GRPCWEB_DEVTOOLS__";
+
+export const grpcWebDevToolsInterceptor: RpcInterceptor = {
+    interceptUnary(next, method, input, options) {
+        const res = next(method, input, options);
+        // @ts-expect-error Variable injected by grpc-web-devtools in the browser
+        if (window.__GRPCWEB_DEVTOOLS__) {
+            const methodType = "unary";
+            const m = `${method.service.typeName}/${method.name}`;
```

Comment:
```suggestion
            const method = `${method.service.typeName}/${method.name}`;
```

I saw something in the comment you referenced about the length of this variable. Is there a problem naming it `method`? Or what exactly is the reason to only call it `m`?

Reply:
`method` cannot be used as it is already a parameter name. I've named it `methodName`, which suits it quite well imo.